src/components/AddProductDialog.vue

```diff
@@ -111,18 +123,56 @@
                     </v-window-item>
                 </v-window>
             </div>
+            <div class="d-flex flex-column align-center justify-end pa-4">
+                <v-alert
```

Comment:
While not part of this issue, long term, I would highly recommend working on a more general error/notification system, to not have to add sth like this to almost every page.
Potential solutions could include snackbars, a non-fullscreen dialog, ...

Reply:
I agree. I did already mention this in the corresponding Gropius issue and I will add a dedicated issue for this.